src/dcp/dcp.c

```diff
@@ -82,6 +82,7 @@ void print_usage(void)
 #ifdef DAOS_SUPPORT
     printf("      --daos-prefix        - DAOS prefix for unified namespace path\n");
     printf("      --daos-api           - DAOS API in {DFS, DAOS} (default uses DFS for POSIX containers)\n");
+    printf("      --daos-preserve      - preserve DAOS container properties and user attributes\n");
```

Comment:
@dsikich
1. Should we also wrap this in #ifdef HDF5_SUPPORT?
2. Or maybe at least mention HDF5 in the help text?
3. We should mention that a path is expected so the user doesn't think this is just a flag.
4. We should probably also update the doc: https://github.com/hpc/mpifileutils/blob/master/doc/rst/dcp.1.rst